lms/services/jstor/_article_metadata.py

```diff
@@ -47,7 +56,25 @@ def from_response(cls, response):
         return cls(_ArticleMetadataSchema(response).parse())
 
     def as_dict(self):
-        return {"title": self.title, "content_status": self.content_status}
+        return {
+            "title": self.title,
```

Comment:
Doesn't "title" need to call `_get_title(self, title_field, subtitle_field)` to potentially append the subtitle?

Looks like _ArticleMetadataSchema has title/subtitle both at the article and book levels.

Reply:
Yep, this happens here:

 * https://github.com/hypothesis/lms/blob/d401fa8ef7e0ffd053c3df3ebd0ed4ca2bc01c1b/lms/services/jstor/_article_metadata.py#L102

Reply:
Grr GH diff view tricked me, looks good :+1: